pkg/plugins/mavenmetadata/main.go

```diff
@@ -0,0 +1,90 @@
+package mavenmetadata
+
+import (
+	"encoding/xml"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/sirupsen/logrus"
+	"github.com/updatecli/updatecli/pkg/core/httpclient"
+	"github.com/updatecli/updatecli/pkg/core/result"
+)
+
+// DefaultHandler is the default implementation for a maven metadata handler
+type DefaultHandler struct {
+	metadataURL string
+	webClient   httpclient.HTTPClient
+}
+
+// New returns a newly initialized DefaultHandler object
+func New(metadataURL string) *DefaultHandler {
+	return &DefaultHandler{
+		metadataURL: metadataURL,
+		webClient:   http.DefaultClient,
+	}
+}
+
+// getMetadatas is an internal method that returns the parsed metadata object
```

Comment:
```suggestion
// getMetadata is an internal method that returns the parsed metadata object
```
The "s" seems weird to me because we only get "one" metadata file content

Reply:
- You're correct, it is "1 metatdata file"
- But we retrieves the (multiple) metadataS from the file (e.g. the list of versions)
=> hence the plural form for the private method.

(Good catch for the comment thoug, I'm fixing it now)

Reply:
Given what this method does, WDYT renaming it to `getMetadataFile()` ? that would make it clearer and closer to the singular form you are proposing?